src/Template/Adrs/xml/e2b.ctp

```diff
@@ -43,7 +43,7 @@
         ?></additionaldocument>
         <documentlist><?php
             foreach ($adr['attachments'] as $attachment):
-                echo $attachment['description'].', ';
+                echo preg_replace('/[^A-Za-z0-9\s+]/', '',$attachment['description']).', ';
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate